src/components/entity/table-view.vue

```diff
@@ -0,0 +1,155 @@
+<!--
+Copyright 2025 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <entity-table v-show="odataEntities.dataExists" ref="table"
+    v-model:all-selected="allSelected" :properties="dataset.properties"
+    :deleted="deleted" :awaiting-deleted-responses="awaitingResponses"
+    v-on="reemitters"/>
```

Comment:
I really like `v-on="reemitters"` pattern, can we paste it in SubmissionTableView as well